src/bundle/Controller/LocationController.php

```diff
@@ -62,4 +64,26 @@ public function actionsAction(
 
         return $this->reloadTab('locations', $content->id, $redirectLocationId);
     }
+
+    public function updateLocationDefaultSortOrderAction(
+        Location $location,
+        Content $content,
+        Request $request,
+        LocationService $locationService,
+        UiFormFactory $formFactory
+    ) {
+        $form = $formFactory->createLocationOrderingForm($location);
+        $form->handleRequest($request);
+
+        if ($form->isValid()) {
+            $updateStruct = $locationService->newLocationUpdateStruct();
```

Comment:
Could the `data_class` of the form be set to `LocationUpdateStruct` to avoid this?